.circleci/config.yml

```diff
@@ -1,30 +1,39 @@
 version: 2
 jobs:
-  build:
+  test:
     docker:
-      - image: circleci/python:3.6.1
-    working_directory: ~/repo
+      - image: circleci/python:3.6.4
+      - image: trufflesuite/ganache-cli:v6.4.1
+        command: -d -k=petersburg -i 1001
+    working_directory: ~/build
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "requirements.txt" }}
-            - v1-dependencies-
       - run:
-          name: install dependencies
-          command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install -r requirements.txt
+          name: Install Dependencies
+          command: sudo pip install -r requirements.txt
+      - run:
+          name: Test
+          command: tox
       - save_cache:
```

Comment:
Ok, this is still not the same pattern we have in all our other repo, but I guess it's fine for now. In all other repos we have one task that just installs then saves the cache, then we do all the other ones in parallel which load this cache.

Here you're duplicating the install (doing it on each task). It's fine for now if you want to do it this way I suppose (though it's weird it's not the same as everything else). If you do it this way though there is no need to `save_cache`

Did having an install step and saving that cache and then loading it just not work or something?

Reply:
doing it cacheless for now